hazelcast/src/test/java/com/hazelcast/internal/management/operation/DisabledOperationsTest.java

```diff
@@ -0,0 +1,96 @@
+import com.hazelcast.client.impl.spi.impl.ClientInvocationFuture;
+import com.hazelcast.client.test.TestHazelcastFactory;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.annotation.ParallelJVMTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.security.AccessControlException;
+import java.util.concurrent.ExecutionException;
+
+import static com.hazelcast.test.HazelcastTestSupport.ASSERT_TRUE_EVENTUALLY_TIMEOUT;
+import static com.hazelcast.test.HazelcastTestSupport.randomString;
+import static com.hazelcast.test.HazelcastTestSupport.smallInstanceConfig;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class DisabledOperationsTest {
+
+    HazelcastClientInstanceImpl client;
+    HazelcastInstance member;
+    private TestHazelcastFactory factory;
+
+    @Before
+    public void setUp() {
+        factory = new TestHazelcastFactory();
+
+        // scripting and console are disabled by default
+        member = factory.newHazelcastInstance(smallInstanceConfig());
+        client = ((HazelcastClientProxy) factory.newHazelcastClient()).client;
+    }
+
+    @After
+    public void tearDown() {
+        factory.shutdownAll();
+    }
+
+    @Test
+    public void testRunConsoleCommandIsNotAllowed() throws Exception {
+        ClientMessage clientMessage = MCRunConsoleCommandCodec.encodeRequest(randomString(), randomString());
+        assertFailure(clientMessage, "Using Console is not allowed on this Hazelcast member.");
+    }
+
+    @Test
+    public void testRunScriptIsNotAllowed() throws Exception {
+        ClientMessage clientMessage = MCRunScriptCodec.encodeRequest(randomString(), randomString());
+        assertFailure(clientMessage, "Using ScriptEngine is not allowed on this Hazelcast member.");
+    }
+
+    private void assertFailure(ClientMessage clientMessage, String expectedExceptionMsg) throws Exception {
+        ClientInvocation invocation = new ClientInvocation(client, clientMessage, null);
+        ClientInvocationFuture future = invocation.invoke();
+        try {
+            future.get(ASSERT_TRUE_EVENTUALLY_TIMEOUT, SECONDS);
+            fail("Execution was successful whereas failure was expected.");
+        } catch (ExecutionException e) {
+            Throwable cause = e.getCause();
+            assertTrue("Cause is of type " + cause.getClass().toString(),
+                    cause.getClass().isAssignableFrom(AccessControlException.class));
+            assertEquals(expectedExceptionMsg, cause.getMessage());
+        }
```

Comment:
Maybe use assertj instead? 

```suggestion
        Assertions.assertThatThrownBy(() -> future.get(ASSERT_TRUE_EVENTUALLY_TIMEOUT, SECONDS))
                .hasCauseInstanceOf(AccessControlException.class)
                .hasRootCauseMessage(expectedExceptionMsg);
```